pkg/cmd/bootstrap.go

```diff
@@ -39,6 +39,19 @@ const (
 	pipelinesOperatorName  = "OpenShift Pipelines Operator"
 )
 
+// Default flag values
+var defaultFlagValues = map[string]interface{}{
```

Comment:
Since all the map values are of type string can we change the value type to string? 
```suggestion
var defaultFlagValues = map[string]string{
```